src/net-to-gross/netToGross.js

```diff
@@ -0,0 +1,55 @@
+const Big = require('big.js');
+const todayDate = new Date();
+const calculateTaxes = require('../taxCalculator.js');
+const insurancesCalcGross = require('./calcInsurance.js');
+const calculateTaxableIncome = require('./calcTaxableIncome.js');
+const insurancesCalculator = require('../insurancesCalculator.js');
+
+module.exports = async function netToGross(net, dependentCount = 0, region = 1, date = todayDate) {
+
+    const netBig = new Big(net)
+    if (netBig.lte(0)) {
+        return 0
```

Comment:
Is return 0 a good idea? We agreed that this function should return object of a payslip